__tests__/app.test.js

```diff
@@ -198,4 +198,36 @@ describe("7. GET /api/articles/:article_id (comment count)",() => {
       })
     })
   });
+});
+
+describe.only("8. GET /api/articles", () => {
+  test('Status 200: responds with an array of objects contaning all articles. Each article object should include additional "votes" and "comment_count" property', () => {
+  return request(app)
+  .get("/api/articles")
+  .expect(200)
+  .then(({body: {articles}}) => {
+      expect(articles).toHaveLength(12)
+      articles.forEach((article) => {
+        expect.objectContaining({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
```

Comment:
The get all articles endpoint, should not supply a body

Reply:
Good spot - thank you! Will remove it now!